### ISSUE TITLE
copyrights/licenses check was removed, probably by mistake

### ISSUE THREAD

**Issue by yegor256** [objectionary/eo-phi-normalizer#628]
#535 was closed, but `copyrights.yml` is not in `.github/workflows` any more. Any particular reason for disabling this check or it was removed by mistake?

**Comment by yegor256** [comment]
@deemp please, check

**Comment by deemp** [comment]
> Many smaller files will make build easier to understand for contributors: why it fails.

@yegor256, we want to make contributors fix all problems discovered during static checks before running the tests and the pipeline. The most straightforward way to achieve this goal is to make the `tests` and `pipeline` jobs depend on the `check` job.

![image](https://github.com/user-attachments/assets/447acfb5-83cb-4932-990e-dde78ca6b840)

Usually, it's easy to discover the problem if one of the static checks fails. 

Here, none of them has failed.

![image](https://github.com/user-attachments/assets/418c1cdd-2c71-4ba3-a7bd-b91846c03320)

**Comment by deemp** [comment]
@yegor256, to maintain the code quality. Citing ChatGPT:

```
If static checks fail, there's no point in running tests. Running tests on code that has basic quality issues or linting violations would be a waste of CI resources, as the code is already in an unacceptable state. Running static checks first allows you to catch these issues early and avoid unnecessary test runs.
```

**Comment by yegor256** [comment]
@deemp fair enough. Can you make jobs depend on each other while staying in different YAML files?
